Documentation/Elastics/ElasticSystem.md

```diff
@@ -0,0 +1,76 @@
+
+Currently elastics manager offers configuration objects options for the following transform types, each of them represented by a [elastic configuration object](#elastic-configuration-object):
+- Translation Elastic
+- Rotation Elastic
+- Scale Elastic
+
+#### Elastic configuration object
+A elastics configuration defines properties for a damped harmonic oscillator differential system. 
+The following properties can be adjusted but already come with a set of defaults in MRTK:
+- **Mass**: mass of the simulated oscillator element.
+- **HandK**: hand spring constant.
+- **EndK**: end cap spring constant.
+- **SnapK**: snap point spring constant.
+- **Drag**: drag/damper factor, proportional to velocity.
+
+### Elastics extents
+Elastics extents settings vary depending on the type of manipulation. Translation and scale are represented by [volume elastic extents](#volume-elastic-extent) where rotation is represented by a [quaternion elastic extent](#quaternion-elastic-extent).
+
+#### Volume elastic extent
+Volume extents define a three dimensional space in which the damped harmonic oscillator is free to move.
+
+![Elastic Volume Stretch Bounds](../Images/Elastics/Elastics_Volume_Bounds.gif)
+
+- **StretchBounds**: represents the lower bounds of the elastic space.
+- **UseBounds**: whether the stretch bounds should be respected by the system. If set to true if the current iteration of the target position is outside the stretch bounds, the end force will be applied.
+- **SnapPoints**: points inside the space to which the system will snap.
+- **RepeatSnapPoints**: repeats the snap points to infinity. Existing snap points will serve as a modulo where the actual snap points are mapped to the closest integer multiples of every snap point.
+- **SnapRadius**: distance at which snap points begin forcing the spring.
+
+![Elastic Volume Snap Grid](../Images/Elastics/Elastics_Volume_Snap.gif)
+
+#### Quaternion elastic extent
+Quaternion extents define a four dimensional rotation space in which the damped harmonic oscillator is free to rotate.
+
+![Elastic Rotation Example](../Images/Elastics/Elastics_Rotation.gif)
+
+- **SnapPoints**: euler angles to which the system will snap.
+- **RepeatSnapPoints**: repeats the snap points to infinity. Existing snap points will serve as a modulo where the actual snap points are mapped to the closest integer multiples of every snap point.
+- **SnapRadius**: arc-angle at which snap points begin forcing the spring in euler degrees.
+
+## Elastics example scene
+
+You can find examples of elastics configurations in the `ElasticSystemExample` scene.
+
+<img src="../Images/Elastics/Elastics_Example_Scene.png">
```

Comment:
Why does this line use <img> but other ones uses the markdown format:

```
![Elastic Rotation Example](../Images/Elastics/Elastics_Rotation.gif)
```

Reply:
because i copied it from a file that uses html tags instead of markdown format :D - changing